### DISCUSSION TITLE
I want to set up essentials anti build for one world.

### DISCUSSION THREAD

**Discussion by eurps** [EssentialsX/Essentials#5767] [Q&A] [answered]
I want to set up essentials anti build for one world and a designated spawn area. Does anyone know how?

**Comment by ImDarkLaw** [comment, marked as answer]
> I want to set up essentials anti build for one world and a designated spawn area. Does anyone know how?

EssentialsAntiBuild doesn't include any kind of region protection; to restrict building in specific regions we recommend installing [WorldGuard](https://enginehub.org/worldguard) instead.
